### ISSUE TITLE
sequence to sequence one dimense CNN 

### ISSUE THREAD

**Issue by cswwp** [keras-team/keras#7652]
i want to build a sequence to sequence one-dimense CNN model, my output include 30 state, for instance: the outout is [  [0,1,3,5],
                        [0,2,4,1],
                        .........
                    ] 
each row presents one sample, and every sample has 4 state.
but i don't know how to write the cnn output softmax layer, i try to write like this ,but it doesn't work:

model.add(Conv1D(filters,
                     kernel_size,
                     padding='valid',
                     activation='relu',
                     strides=2, input_shape=[687, 1]))
    # we use max pooling:
    model.add(MaxPooling1D())
    model.add(Conv1D(filters,
                     kernel_size,
                     padding='valid',
                     activation='relu',
                     strides=1,))
    model.add(GlobalMaxPooling1D())
    # We add a vanilla hidden layer:
    model.add(Dense(hidden_dims))
    model.add(Dropout(0.2))
    model.add(Activation('relu'))
    # We project onto a single unit output layer, and squash it with a sigmoid:
    model.add(Dense(11, output_dim=3, activation='softmax'))

please help me

**Comment by cswwp** [comment]
it shows that output_dim isn't exist, and my keras version is 2.0.2, the backend is tensorflow, i expect my output is 3 dimenseion like this [sample, depth, 11 ], depth is the one_hot depth
